### PR TITLE
Add environment variables to docker run cmd

### PR DESCRIPTION
**Pull request by galindro**
Fixes https://github.com/UnitedIncome/serverless-python-requirements/issues/230

### CODE REVIEW

**Comment by AndrewFarley** [comment]
Haha... uhh @galindro you cloned "my" fork, and tried to MR it here.  If you can, re-fork from this codebase...
```
git remote add upstream https://github.com/UnitedIncome/serverless-python-requirements.git
git reset --hard upstream/master
```
Then patch your changes, and force push so we get just your feature here.

And secondarily, I'd suggest making a test (look in test.bats) to prove/ensure this feature works, and continues to work as other changes occur.  Thanks!

**Comment by galindro** [comment]
I've forked your repo because I've did 1 enhancement + 1 bug fix in the same PR. The enhancement can be applied to current UnitedIncome/serverless-python-requirements master branch but the fix no. It depends from your fork.

Therefore, what can I do is:

1. Update this PR with a single commit to add the docker environment feature
2. Create a PR **in your fork** to fix the problem that I've mentioned with the command executed in docker run: replace `&&` by `;`. This way you can push a new version of it to your already open PR: https://github.com/UnitedIncome/serverless-python-requirements/pull/165

What do you think?

**Comment by dschep** [comment]
Hey @galindro, I've merged @AndrewFarley's PR, you should be able to rebase on master now :smiley:

**Comment by galindro** [comment]
Done @dschep . I took advantage to make the two changes aforementioned [here](https://github.com/UnitedIncome/serverless-python-requirements/pull/231#issuecomment-418444454)

File: lib/pip.js [hunk 1]
```diff
@@ -210,7 +219,7 @@ function installRequirements(targetFolder, serverless, options) {
           ])
         );
       }
-      pipCmd = ['/bin/bash', '-c', '"' + commands.join(' && ') + '"'];
+      pipCmd = ['/bin/bash', '-c', '"' + commands.join(' ; ') + '"'];
```
Comment (dschep) [inline lib/pip.js]: What's the motivation for changing this? Would you really want to keep running commands if one of them fails?

**Comment by miketheman** [comment]
#Triage 
This PR has a bit of history to it, and needs to be reconfirmed whether the original issue reported remains a problem.
@AndrewFarley you had some historical context on this?

**Comment by AndrewFarley** [comment]
This indeed still looks useful, and it's quite a minor patch, could use a rebase and additions to the README, and perhaps a test.

I'll take this and get it prepared for merge @miketheman

**Comment by miketheman** [comment]
This appears to have been resolved in #271. Closing!

**Pull request closed without merging**